db/schema.rb

```diff
@@ -120,6 +130,20 @@
     t.index ["wf_stage"], name: "index_holdings_on_wf_stage"
   end
 
+  create_table "holdings_institutions", id: false, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
```

Comment:
Maybe I'm missing something, but we already have a `holdings_to_institutions` table...